.github/workflows/docs-lint.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Lint Docs
+
+on:
+    push: ~
+    pull_request: ~
+
+jobs:
+  docs-lint:
+    name: Markdownlint
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: github/super-linter@v3
+        env:
+          DEFAULT_BRANCH: latest
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          LINTER_RULES_PATH: '.' # From https://github.com/github/super-linter/pull/859#issuecomment-709625090
+
+          VALIDATE_MARKDOWN: true
+          MARKDOWN_CONFIG_FILE: docs/.markdownlint.yml
+          FILTER_REGEX_INCLUDE: '^docs/(1\.5|2\.0)/.*'
```

Comment:
Can we also lint the Markdown files that live outside the version-specific directories?  If that means we also have to lint the older versions too I'd be okay with that.

Reply:
Yes, I'll pick that up next week. Do you want it on only docs or the README and root project markdown files?

Reply:
Great question! I think just the docs - I don't mind if the README, CHANGELOG, etc. are slightly different.  And we don't want to lint any of the Markdown files used in our tests.

No rush on my end.  I appreciate your help here!

Reply:
@colinodell All of `docs/` is now linted and all errors have been resolved.